apps/financial-aid/backend/src/app/modules/application/application.service.ts

```diff
@@ -136,10 +136,10 @@ export class ApplicationService {
     await this.sendEmail(
       {
         name: user.name,
-        address: application.email,
+        address: appModel.email,
       },
       appModel.id,
-      `Umsókn þín er móttekin og er nú í vinnslu.`,
+      `Umsókn þín er móttekin og er nú í vinnslu. <a href="https://fjarhagsadstod.dev.sveitarfelog.net/stada/${appModel.id}" target="_blank"> Getur kíkt á stöðu síðuna þína hér</a>`,
```

Comment:
The base url here should be a env variable in the future, not needed right now just an fyi